swagger/v1/openapi.json

```diff
@@ -5,8 +5,9 @@
     "version": "v1",
     "description": "This is the API for Cloud Services for RHEL Compliance. You can find out more about Red Hat Cloud Services for RHEL at [https://cloud.redhat.com/](https://cloud.redhat.com/)"
   },
+  "basePath": "/api/compliance",
```

Comment:
Why do we need this one here? I checked spec with this raw and validator is complaining
`Structural error at 
should NOT have additional properties
additionalProperty: basePath
Jump to line 0`

Reply:
Thanks for checking! I found it in approval's openapi.json, but I think the better way to specify this is via `servers`. I'll update it.